### PR TITLE
add beancount.io iOS and Android Apps

### PR DESCRIPTION
**Pull request by puncsky**

### CODE REVIEW

**Comment by siddhantgoel** [comment]
Since these are commercial apps, could you please create a section called "Commercial Apps" and move these two entries there?

**Comment by puncsky** [comment]
> Since these are commercial apps, could you please create a section called "Commercial Apps" and move these two entries there?

sure, thanks for reviewing!

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@
 
 - [Beancount.io](https://beancount.io/) - Out-of-box Beancount and Fava SaaS that you can use instantly
 
+## Commercial Apps
+
+- [Beancount.io iOS App](https://apps.apple.com/us/app/id1527950512) - First-ever App on iOS that visualizes and connects to your Beancount ledger elegantly
+- [Beancount.io Android App](https://play.google.com/store/apps/details?id=io.beancount.android) - Add a transaction to your Beancount ledger quickly
```
Comment (siddhantgoel) [inline README.md]: Is it possible to reframe the descriptions here to something neutral?

"First-ever App on iOS that visualizes and connects to your Beancount ledger elegantly" sounds a bit advertise-y and ideally I'd like to keep everything on awesome-beancount.com neutral.

One suggestion would be "iOS app for Beancount.io", and the same for the Android app. What do you think?

**Review by siddhantgoel — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -42,6 +42,11 @@
 
 - [Beancount.io](https://beancount.io/) - Out-of-box Beancount and Fava SaaS that you can use instantly
 
+## Commercial Apps
+
+- [Beancount.io iOS App](https://apps.apple.com/us/app/id1527950512) - Visualizes and connects to your Beancount ledger on iOS
```
Comment (siddhantgoel) [inline README.md]:
```suggestion
- [Beancount.io iOS App](https://apps.apple.com/us/app/id1527950512) - Visualizes and connects to your Beancount.io ledger on iOS
```

File: README.md [hunk 3]
```diff
@@ -42,6 +42,11 @@
...
+- [Beancount.io Android App](https://play.google.com/store/apps/details?id=io.beancount.android) - Visualizes and connects to your Beancount ledger on Android
```
Comment (siddhantgoel) [inline README.md]:
```suggestion
- [Beancount.io Android App](https://play.google.com/store/apps/details?id=io.beancount.android) - Visualizes and connects to your Beancount.io ledger on Android
```

**Review by siddhantgoel — CHANGES_REQUESTED** [review]

**Pull request merged**